# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/apz/util/ChromeProcessController.cpp
# Commit: e974d872becd
# Full Hash: e974d872becd4fae0214738cdf27173a994d0f59
# Author: Dan Robertson <drobertson@mozilla.com>
# Date: 2023-08-30 21:27:31
# Regressor Bug: 1816473
# File Overlap Count: 1
# Description:
#   Bug 1816473 - Do not delay dispatching synthesized mouse events. r=botond,edgar
#   
#   Do not delay dispatching synthesized mouse events for a single tap
#   gesture.
#   
# ==============================================================================

diff -r 1b272e14e402 -r e974d872becd gfx/layers/apz/util/ChromeProcessController.cpp
--- a/gfx/layers/apz/util/ChromeProcessController.cpp	Wed Aug 30 12:06:41 2023 +0000
+++ b/gfx/layers/apz/util/ChromeProcessController.cpp	Wed Aug 30 12:49:22 2023 +0000
@@ -94,6 +94,9 @@
 
   MOZ_ASSERT(mUIThread->IsOnCurrentThread());
   mWidget = nullptr;
+  if (mAPZEventState) {
+    mAPZEventState->Destroy();
+  }
   mAPZEventState = nullptr;
 }
 
